packages/dd-trace/src/appsec/sdk/track_event.js

```diff
@@ -2,12 +2,27 @@
 
 const log = require('../../log')
 const { getRootSpan } = require('./utils')
-const { setUserTags } = require('./set_user')
+const { setUserTagsSdk } = require('./set_user')
 const waf = require('../waf')
 const { keepTrace } = require('../../priority_sampler')
 const addresses = require('../addresses')
 const { ASM } = require('../../standalone/product')
+const telemetryMetrics = require('../../telemetry/metrics')
 
+const appsecMetrics = telemetryMetrics.manager.namespace('appsec')
+
+function increaseSdkEventMetric (eventType, version) {
+  const tags = {
+    event_type: eventType,
+    sdk_version: version
+  }
+
+  appsecMetrics.count('sdk.event', tags).inc(1)
+}
+
+/**
+ * @deprecated in favor of trackUserLoginSuccessV2
+ */
```

Comment:
not sure those are super useful as those functions are not used anywhere really. We have proxy functions that are used for the SDK insteads, and you've already set them as deprecated in the typings file, i think that's enough, no need it here too

Reply:
don't you think it adds value when in the future, when you want to modify the new (or old) method to find easily what is the one that need to be modified?